src/components/custom/other-platforms.js

```diff
@@ -44,9 +44,9 @@ const StyledSubHeader = styled(Header)`
     max-width: ${props => props.maxWidth || ''};
 `
 
-export const OtherPlatform = ({ header, subHeader, exclude }) => (
-    <SectionContainer>
-        <HeaderWrapper>
+export const OtherPlatform = ({ header, subHeader, exclude, nav }) => (
+    <SectionContainer padding='0'>
+        {nav ? null : (<HeaderWrapper>
```

Comment:
is it to check if its a nav?, if it is, please change the name to is_nav

Reply:
done